src/components/UserDetailSections/UserAccounts/UserAccounts.js

```diff
@@ -60,17 +61,20 @@ const UserAccounts = ({
   const displayWhenOpen = (<Button disabled={buttonDisabled} to={{ pathname: `/users/${params.id}/charge` }}><FormattedMessage id="ui-users.accounts.chargeManual" /></Button>);
 
   useEffect(() => {
+    const refundStatusesValues = Object.values(refundStatuses);
+    const feeFineActions = get(resources, ['feefineactions', 'records'], []);
```

Comment:
Yeah, I know about this syntax. Not sure why I used `get` here instead. While switching from one module to another I see that in some `get` is used and in some `.?` is used. I'll replace.

But I did not know about `??`. If it is in new proposal then why they decided to add new `??` operator if we can use just `||` operator. I'll read about this. Thank you.

Reply:
> This change looks fine to me, but without any comments (I realize there are none in the original) it is quite opaque. In an ideal world, it would be accompanied by a unit test as well.

I'll add comments as well.